docs/cloud-native-security/index.rst

```diff
@@ -4,14 +4,10 @@
 Cloud Native Security
```

Comment:
Cloud-Native Security

Reply:
"Cloud Native Security" is the official name of the offer, so we need to keep it like that.

Reply:
Understood!!